dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -916,20 +916,20 @@ BEGIN
     IDS_SHELL_ABOUT_BACK "< Înap&oi"
     FCIDM_SHVIEW_NEW "&Crează" /* A menu item with an ampersand */
     FCIDM_SHVIEW_NEWFOLDER "&Folder"
-    FCIDM_SHVIEW_NEWLINK "Sc&urtătură"
+    FCIDM_SHVIEW_NEWLINK "&Comandă rapidă"
     IDS_FOLDER_OPTIONS "Opțiuni foldere"
     IDS_RECYCLEBIN_LOCATION "Locația pentru Coșul de reciclare"
     IDS_RECYCLEBIN_DISKSPACE "Spațiul disponibil"
-    IDS_EMPTY_BITBUCKET "Golește Coșul de reciclare"
-    IDS_PICK_ICON_TITLE "Alegerea pictogramei"
+    IDS_EMPTY_BITBUCKET "Golire Coş de reciclare"
+    IDS_PICK_ICON_TITLE "Alegere pictogramă"
     IDS_PICK_ICON_FILTER "Fișiere pictogramă (*.ico, *.icl, *.exe, *.dll)\0*.ico;*.icl;*.exe;*.dll\0Orice fișier (*.*)\0*.*\0"
-    IDS_OPEN_WITH_FILTER "Fișiere executabile (*.exe)\0*.exe\0Orice fișier (*.*)\0*.*\0"
+    IDS_OPEN_WITH_FILTER "Programe (*.exe)\0*.exe\0Toate fișierele (*.*)\0*.*\0"
```

Comment:
```suggestion
    IDS_OPEN_WITH_FILTER "Programe (*.exe)\0*.exe\0Toate fișier (*.*)\0*.*\0"
```
proposed for consistency with IDS_RUNDLG_BROWSE_FILTER, I do hope it is correct from a grammar pov.